pkgs/development/python-modules/textual/default.nix

```diff
@@ -92,6 +92,8 @@ buildPythonPackage rec {
       # https://github.com/Textualize/textual/issues/5327
       "test_cursor_page_up"
       "test_cursor_page_down"
+
+      "test_async_reactive_watch_callbacks_go_on_the_watcher"
```

Comment:
The error message shows that the test fails on Python3.12, but this set of ignored tests is only added for Python >= 3.13, so I don't think this is correct.

Furthermore I'm not sure that this is the correct list to add the ignored test to (assuming you mean to show that issue 5327 is the root-cause?)
The error messages in that issue don't match this one (RuntimeError: There is no current event loop in thread 'MainThread'.)
I think issue 5511 would be a better match; you'd have to add a new list for that, if you wish to indicate the cause of the failing test. I'm open to correction ofc :)

Reply:
Why do we think that disabling `test_async_reactive_watch_callbacks_go_on_the_watcher` is the fix for this issue? I was able to build the package on darwin, with the same warning occurring in the test output.

Can you post more of the logs you got from textual failing to build on your system?